src/classes/project_data.py

```diff
@@ -288,6 +288,8 @@ def new(self):
                     self._data["width"] = profile.info.width
                     self._data["height"] = profile.info.height
                     self._data["fps"] = {"num" : profile.info.fps.num, "den" : profile.info.fps.den}
+                    self._data["display_ratio"] = {"num": profile.info.display_ratio.num, "den": profile.info.display_ratio.den}
+                    self._data["pixel_ratio"] = {"num": profile.info.pixel_ratio.num, "den": profile.info.pixel_ratio.den}
```

Comment:
This nonsense (manually constructing the dict) actually sent me off on _quite_ the tangent today, and resulted in OpenShot/libopenshot#321. Which may hopefully eliminate the need to do that, eventually. But, for now, it is indeed necessary.